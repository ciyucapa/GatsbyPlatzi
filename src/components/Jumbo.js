import React from 'react';
import PropTypes from "prop-types";

import {StyledJumbo} from '../styles/components';

const Jumbo = (props) => (
    <StyledJumbo>
        <div style={{display: "flex", flexDirection: "column"}} >
            <h2>!Consigue el mejor swag exclusivo de PlatziÂ¡</h2>
            <small>{props.description}</small>
        </div>
        {props.children}
    </StyledJumbo>
);

Jumbo.defaultProps = {
    description: '',
}
  
Jumbo.propTypes = {
    description: PropTypes.string,
}

export default Jumbo;