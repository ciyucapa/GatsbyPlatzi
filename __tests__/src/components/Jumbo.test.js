import React from "react"
import renderer from "react-test-renderer"
import Jumbo from '../../../src/components/Jumbo'

describe("ImageComponente", () => {
    it("renders correctly", () => {
      const tree = renderer
        .create(<Jumbo 
                  description="El mejor sitio" 
                >
                  <div>Test</div>
                </Jumbo>
                  )
        .toJSON()
      expect(tree).toMatchSnapshot()
    });
});