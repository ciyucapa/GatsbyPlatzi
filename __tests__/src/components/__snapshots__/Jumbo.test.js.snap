// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageComponente renders correctly 1`] = `
<div
  className="sc-bwzfXH kkAmTB"
>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
      }
    }
  >
    <h2>
      !Consigue el mejor swag exclusivo de PlatziÂ¡
    </h2>
    <small>
      El mejor sitio
    </small>
  </div>
  <div>
    Test
  </div>
</div>
`;
